app/stac_api/models.py

```diff
@@ -36,22 +36,13 @@
 # instance so that it's not shared between all field instances.
 # HINT: Use a callable instead, e.g., use `list` instead of `[]`.""
 # <end quote>
-DEFAULT_STAC_EXTENSIONS = {
-    "EO": "eo",
-    "PROJ": "proj",
-    "VIEW": "view",
-    "GEOADMIN-EXTENSION": "https://data.geo.admin.ch/stac/geoadmin-extension/1.0/schema.json"
-}
-
 DEFAULT_EXTENT_VALUE = {"spatial": {"bbox": [[None]]}, "temporal": {"interval": [[None, None]]}}
 
 DEFAULT_SUMMARIES_VALUE = {"eo:gsd": [], "geoadmin:variant": [], "proj:epsg": []}
 
 
-def get_default_stac_extensions(is_collection=False):
-    if is_collection:
-        return list()
-    return list(DEFAULT_STAC_EXTENSIONS.values())
+def get_default_stac_extensions():
+    return list()
```

Comment:
this could maybe be removed as well (since afaik it's only called in one place). It's still reference in a migration, this would be a rare case where modifying old migrations is acceptable (replace the function name in the migrations with just `list`)

Reply:
Yes, this could be removed. It only was introduced because once upon a time the stac_extensions were stored in an ArrayField I think. And hence it was required to have a callable to set the default values. But now implementation has changed.

I only left the whole construct with the two functions (one in the `models.py` and one in `serializers.py`) just for the case, that in future we might want to have non-empty stac_extensions again. But it is clearer to delete it for now and re-implement in case we need it in future.

Reply:
:+1:

Reply:
done